portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,41 @@
+body {
+    background-image: url("images/background.png");
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-attachment: fixed;
+    background-position: center center;
```

Comment:
nit: Isn't this the same as center?

Reply:
I am not sure what do you mean by this?

Reply:
Just that "background-position: center;" should work equally well?

"background-position: center center;" feels a bit redundant.